### PR TITLE
Add access by user login to sitetrees menu

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/admin/sitetrees.py [hunk 1]
```diff
@@ -39,63 +39,75 @@
                 'data_center.change_virtualserver'
             ],
             perms_mode_all=False,
+            access_loggedin=True,
```
Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: what do you think about creating some `ralph_item` which will be partial to item with `access_loggedin=True` set?

Comment (szok) [inline src/ralph/admin/sitetrees.py]: good idea

File: src/ralph/admin/sitetrees.py [hunk 2]
```diff
@@ -21,9 +21,15 @@
 
 activate(settings.LANGUAGE_CODE)
 
+
+def ralph_item(*args, **kwargs):
+    kwargs['access_loggedin'] = True
```
Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: maybe use `setdefault` here (is `access_loggedin` is defined, we don't want to overwrite it)

**Comment by ar4s** [comment]
:ok:

**Pull request merged**